src/actions/allTeamsAction.js

```diff
@@ -110,6 +110,7 @@ export const getAllUserTeams = () => {
   return async (dispatch) => {
     userTeamsPromise.then((res) => {
       dispatch(teamMembersFectchACtion(res.data));
+      console.log("getAllUserTeams: res:", res.data)
```

Comment:
Think you should comment your console logs out.  Inspect has like at least 5 or 6 per page load.